src/app/docs/tabs/tabs-docs.component.html

```diff
@@ -4,10 +4,373 @@
+        </ul>
+<!--
+        <div class="sky-text-block">
+            For information about how to use <a href="https://angular.io/docs/ts/latest/cookbook/set-document-title.html">the Angular <sky-code>Title</sky-code> service</a> and the SKY UX <sky-code>title</sky-code> configuration property to specify the title, see <a href="https://developer.blackbaud.com/skyux/learn/reference/configuration#app">the configuration reference</a>.
+        </div>
+-->       
+        
+    </sky-docs-demo-page-section>
+      
+    <sky-docs-demo-page-section
+      heading="Layout"
+    >
+        
+        <sky-docs-design-guideline>
+        <p>
+          Tabs are always the last (bottommost) content in their container. Don’t put additional content under tabs.
+        </p>
+        <sky-docs-design-guideline-thumbnail
+          caption="Don't put additional content under tabs."
+          captionType="danger"                                     
+          imageSource="~/assets/img/guidelines/tabs/tabs-layout-1.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+        
+      <sky-docs-design-guideline>
+        <p>
+          The tab border extends the full width of the container. Don’t use left or right margins on the tab border, and ignore any padding that the container defines.
+        </p>
+        <sky-docs-design-guideline-thumbnail
+          caption="Do extend the tab border to the full width of the container."
+          captionType="success"                                     
+          imageSource="~/assets/img/guidelines/tabs/tabs-layout-2.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+        <sky-docs-design-guideline-thumbnail
+          caption="Don't apply the container's padding to the tab border."
+          captionType="danger"                                     
+          imageSource="~/assets/img/guidelines/tabs/tabs-layout-3.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+        
+      <sky-docs-design-guideline>
+        <p>
+          Tab content does not have additional left or right padding. The left and right gutters align with the gutters for content outside the tabs.
+        </p>
+        <sky-docs-design-guideline-thumbnail
+          caption="Don't apply additional left or right padding to tab content."
+          captionType="danger"                                     
+          imageSource="~/assets/img/guidelines/tabs/tabs-layout-4.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+        
+    </sky-docs-demo-page-section>
+      
+    <sky-docs-demo-page-section
+      heading="Related information"
+    >
+      <sky-docs-design-guideline
+        heading="Components"
+      >
+        
+          <ul>
+              <li><a href="https://developer.blackbaud.com/skyux/components/vertical-tabs">Vertical tabs</a></li>
```

Comment:
Should we add action buttons to this list since we call them out too? Not sure that's necessary, but just checking.